docs/sources/googlecloudstorage.md

```diff
@@ -0,0 +1,65 @@
+# Event Source for Google Cloud Storage
+This Event Source emits Cloudevents when updates occur to a Google Cloud Storage Bucket via a Google [Pub/Sub](https://cloud.google.com/pubsub/docs/overview) Subscription. 
+
+## Prerequisites
+* Google Storage and Pub/Sub enabled in Google Cloud Console
+* Pre-existing [Bucket](https://cloud.google.com/storage/docs/creating-buckets) in Google Cloud Storage
+* A [Service Account](https://cloud.google.com/iam/docs/service-accounts) with rights to Pub/Sub
+* A Pub/Sub Scription created to recieve events on Google Cloud Storage Updates.
+  Example: 
+    ```
+    gsutil notification create -t example -f json gs://<BUCKET>
+    gcloud pubsub subscriptions create bridgedemosub --topic example  
+    ```
+
+
+## Create the Google Google Cloud Storage Source Integration
+
+Create an instance of the Cloud Storage Source at TriggerMesh as part of a Bridge.
+
+- `Name` Is an internal identifier inside the bridge.
+- `project`	Define the Google Cloud Project that the Google Cloud Storage resources live under
+- `subscription` Define the Pub/Sub subscription name 
+
+Save the source, fill the rest of the bridge components and press `Submit Bridge`.
+
+## Events Types
+
+The Cloud Storage Source creates a cloud event for each message received. It creates an event of type `com.triggermesh.googlecloudstorage.event` 
+
+Cloud Event data example:
+
+```json
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: com.triggermesh.googlecloudstorage.event
+  source: io.triggermesh.googlecloudstorage/jeff/gcs
+  id: dcb2ff11-3139-49ee-a9ba-95be25fec854
+  time: 2021-03-26T19:13:10.665004666Z
+  datacontenttype: application/json
+Data,
+  {
+    "kind": "storage#object",
+    "id": "tmdemo/wp5960142-the-great-wave-off-kanagawa-hd-wallpapers.jpg/1616785989346461",
+    "selfLink": "https://www.googleapis.com/storage/v1/b/tmdemo/o/wp5960142-the-great-wave-off-kanagawa-hd-wallpapers.jpg",
+    "name": "wp5960142-the-great-wave-off-kanagawa-hd-wallpapers.jpg",
+    "bucket": "tmdemo",
+    "generation": "1616785989346461",
+    "metageneration": "1",
+    "contentType": "image/jpeg",
+    "timeCreated": "2021-03-26T19:13:09.443Z",
+    "updated": "2021-03-26T19:13:09.443Z",
+    "storageClass": "STANDARD",
+    "timeStorageClassUpdated": "2021-03-26T19:13:09.443Z",
+    "size": "1716402",
+    "md5Hash": "EDETqCAJ5kWxYRL7X/2ykw==",
+    "mediaLink": "https://www.googleapis.com/download/storage/v1/b/tmdemo/o/wp5960142-the-great-wave-off-kanagawa-hd-wallpapers.jpg?generation=1616785989346461&alt=media",
+    "crc32c": "G3Ynzg==",
+    "etag": "CJ3JypTVzu8CEAE="
+  }
+```
```

Comment:
We decided against having that in documentation pages in #109.

Please see @daceynolan's templates in https://github.com/triggermesh/docs/tree/master/templates/source.